### DISCUSSION TITLE
Cannot read properties of undefined (reading 'number')

### DISCUSSION THREAD

**Discussion by stefanlundchristensen** [ObsidianPublisher/obsidian-github-publisher#263] [Q&A — Plugin]
Hi folks,

I have set up github publisher in what I think is the right way (see settings below). However when I try to push things to my repo using the `Upload all shared notes`, I get the below error (see full console in the picture)

` Error with stefanlundchristensen/stefanlundchristensen.github.io: TypeError: Cannot read properties of undefined (reading 'number')`

I can not seem to understand what is going wrong and where this number property is. Any help and direction would be much appreciated!

![image](https://github.com/ObsidianPublisher/obsidian-github-publisher/assets/149314143/64dbc336-099e-49a5-b432-246be3d731dc)

![image](https://github.com/ObsidianPublisher/obsidian-github-publisher/assets/149314143/9db485a7-1884-4f68-a33d-101acccf867d)

**Comment by Mara-Li** [comment]
I won't reply to discussion within bug and that won't follow the guidelines.

**Comment by Mara-Li** [comment]
After searching, I couldn't reproduce the issue.
The error is simple : the plugin can success to make a PR, so its crash after searching (as it should).

I made a little update to prevent crashing at this moment, but can't success to found why your specific repository crash.
Maybe it's a security setting you have enabled that prevent PR on the main branch ?
